packages/passport/sdk/src/magicAdapter.test.ts

```diff
@@ -75,15 +72,27 @@ describe('MagicWallet', () => {
 
       it('does nothing', () => {
         const magicAdapter = new MagicAdapter(config);
-        // @ts-ignore
-        expect(magicAdapter.magicClientPromise).toBeUndefined();
+        // @ts-expect-error: client is private
+        expect(magicAdapter.client).toBeUndefined();
+      });
+
+      it('should throw a browser error for loginWithOIDC', async () => {
+        const magicAdapter = new MagicAdapter(config);
+
+        await expect(async () => {
+          await magicAdapter.login(idToken);
+        }).rejects.toThrow(
+          new PassportError(
+            'Cannot perform this action outside of the browser',
+            PassportErrorType.WALLET_CONNECTION_ERROR,
+          ),
+        );
```

Comment:
`rejects.toThrow` doesnt actually assert the PassportErrorType properly and can be anything and the test will pass. Instead need to do something like:

```
it('should throw a browser error for loginWithOIDC', async () => {
        const magicAdapter = new MagicAdapter(config);

        let type = '';
        let message = '';
        try {
          await magicAdapter.login(idToken);
        } catch (e: any) {
          type = e.type;
          message = e.message;
        }
        expect(type).toEqual(PassportErrorType.WALLET_CONNECTION_ERROR);
        expect(message).toEqual('Cannot perform this action outside of the browser');
      });
```